transcom/mymove
# B-18467: Update service item destination address in updateShipmentDestinationAddress endpoint

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/Default.aspx?menu=StoryBoardPage&feat-nav=--m2)

## Summary

This PR allows the updateShipmentDestinationAddress/createNonSITAddressUpdateRequest update the serviceItems destination address when TOO Approval is not needed on an address change.


### How to test

1. Access MM as prime
2. Update the destination address of a shipment to be updated with the exact same address information to easily pass all checks that will require TOO approval. Prime simulator -> Update shipment destination address.
3. Sign in as TOO
4. There should be no review request for the address change. Look under Move details -> Requested shipments -> edit shipment -> delivery location
6. Confirm the destination SIT service items destination address is the same as the shipments
